tests/globalhelper/runhelper.go

```diff
@@ -35,17 +35,35 @@ func LaunchTests(testSuites []string, skipRegEx string) error {
 		glog.V(5).Info(fmt.Sprintf("set skip regex to %s", skipRegEx))
 	}
 
-	if len(testSuites) > 0 {
+	if len(testSuite) > 0 {
```

Comment:
Is there any use for LunchTests where len(testSuite) == 0 ? If there's no such case, you can just do something like:
```
if testSuite == "" {
    return errors.New("no test suite name provided.")
}
```

Reply:
thanks!